ecs-cli/modules/cli/compose/entity/task/task.go

```diff
@@ -448,6 +461,22 @@ func (t *Task) up(forceUpdate bool) error {
 	return nil
 }
 
+func (t *Task) GetTags() ([]*ecs.Tag, error) {
+	if t.tags == nil {
```

Comment:
I call this function twice. Once to create the RunTaskInput object, and once in entity to create the task def. The first time its called, it will be nil. Then it will be set, and the second time it will not be nil.